pyproject.toml

```diff
@@ -1,3 +1,62 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[tool.flit.module]
+name = "aiida_cp2k"
+
+[project]
+name = "aiida-cp2k"
+dynamic = ["version", "description"]
+readme = "README.md"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: AiiDA",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: MacOS :: MacOS X",
+    "Programming Language :: Python :: 3",
```

Comment:
Good practice to indicate explicitly which Python 3 versions are supported.

Reply:
Let's leave it for the next PR if that is fine.

Reply:
https://github.com/aiidateam/aiida-cp2k/issues/168